src/explore-education-statistics-admin/src/pages/publication/components/__tests__/PublicationDraftReleases.test.tsx

```diff
@@ -210,7 +210,8 @@ describe('PublicationDraftReleases', () => {
           {
             ...testRelease1,
             permissions: {
-              ...testRelease1.permissions,
+              // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+              ...testRelease1.permissions!,
```

Comment:
If we separate the permissions out into a separate variable with an explicit type, we don't need this non-null assertion. Same for below as well.